data/model/organization.py

```diff
@@ -194,3 +195,19 @@ def add_user_as_admin(user_obj, org_obj):
         team.add_user_to_team(user_obj, admin_team)
     except team.UserAlreadyInTeam:
         pass
+
+
+def has_immutable_tags(org_name):
+    """
+    Returns true if the given organization has any repositories containing tags that are set to immutable.
+
+    """
+    return (
+        Tag.select()
+        .join(Repository)
```

Comment:
To find immutable tags inside an organization I have to traverse through the repository, since Tags are linked to Repositories and only those are linked to Users/Organizations. I don't need to explictly use the Repository relation elsewhere in the query since peewee will automatically figure out through the foreign key relationships which fields to use for the join